libnymea-zigbee/backends/deconz/zigbeebridgecontrollerdeconz.cpp

```diff
@@ -233,9 +233,13 @@ void ZigbeeBridgeControllerDeconz::sendNextRequest()
     if (m_currentReply)
         return;
 
-    //    // FIXME: If the controler request queue is full, wait until it's free again
-    //    if (!m_apsFreeSlotsAvailable)
-    //        return;
+    // If the controler request queue is full, wait until it's free again
+    if (!m_apsFreeSlotsAvailable)
+        return;
+
+    if (!m_available) {
```

Comment:
coding style hint... on singleline state with { and one without. According to qt coding style guidlines I'd suggest without...